clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -79,7 +79,7 @@ const TextInput = ({
     setType(type === "password" ? "text" : "password");
 
   return (
-    <InputGroup size="sm" mr="2">
+    <InputGroup size="sm">
```

Comment:
Awesome, thanks for porting this. There might be a wider impact on other forms than when I first did this work but I'm _pretty_ confident it'll be an improvement...